model/DataStore/MetaDataDeliverySyncTask.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoDeliveryRdf\model\DataStore;
+
+use common_exception_Error;
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use core_kernel_persistence_Exception;
+use JsonSerializable;
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\extension\AbstractAction;
+use oat\oatbox\reporting\Report;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\exception\InvalidServiceManagerException;
+use oat\tao\model\metadata\compiler\ResourceJsonMetadataCompiler;
+use oat\tao\model\taskQueue\QueueDispatcher;
+use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use taoQtiTest_models_classes_QtiTestService;
```

Comment:
Looks like a new dependency for this extension

Reply:
This is already a dependency @boajer 

```
    'requires'    => [
        'generis'     => '>=12.32.1',
        'tao'         => '>=45.0.0',
        'taoGroups'   => '>=4.0.0',
        'taoTests'    => '>=12.1.0',
        'taoQtiTest'  => '>=35.4.0',
        'taoDelivery' => '>=13.3.0'
    ],
```